src/lesson8-homework.ts

```diff
@@ -227,39 +248,49 @@ function exercise39() {
     // TODO: get lis of own keys of the user object
     // TODO: iterate over the keys with foreach
     // TODO: console.log the typeof for each property
+    for (let key in user) {
+      console.log(`${key}: ${typeof user[key]}`);
+    }
   }
   printAllUserPropTypes();
 
   // TODO: create function that returns address of the user object,
   // TODO: set the return type of that function using typeof operator
   function getUserAddress() {
-    return null;
+    return user.address;
```

Comment:
set the return type of that function using typeof operator
function getUserAddress(user: TUser): typeof user...